vfabia64/vfabia64.rst

```diff
@@ -1195,6 +1203,19 @@ Note that the ``svbool_t`` parameter is described in `SVE masking`_.
    svfloat32_t _ZGVsM8vv_bar(svfloat64_t vx, svfloat64_t vy,
                              svbool_t vmask);
 
+Streaming-compatible SVE Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The use of ``#pragma omp declare simd`` with ``f``, ``g`` and ``foo``
```

Comment:
How would the compiler know that we are in a streaming or streaming-compatible region?

From what I can find in the ACLE the user adds `__arm_streaming` and `__arm_streaming_compatible` to functions. I can see how this would work in generating calls to streaming compatible vector functions, but I'm struggling with how this would work for generating functions with `#pragma omp declare simd` as these appear to be placed in front of function declarations, which typically won't be within the scope of of a function body.

I'm not at all familiar with OMP though so there could be a way of declaring a streaming compatible region, I've checked https://clang.llvm.org/docs/AttributeReference.html#pragma-omp-declare-simd 

Would be good to give an example if you can?

If this part is somewhat speculative it could be worth separating it from the name mangling part, as I assume someone could manually write the functions rather than have the compiler auto-generate them.